instance/instance.go

```diff
@@ -746,10 +772,14 @@ func handleInstanceErr(ctx context.Context, err error, w http.ResponseWriter, da
 		data = log.Data{}
 	}
 
+	taskErr, isTaskErr := err.(taskErr)
```

Comment:
Variable `taskErr` shadows declaration of struct `taskErr` on line 33.

Maybe use `taskError`?